python/GafferImageTest/ImageWriterTest.py

```diff
@@ -693,7 +708,7 @@ def __testMetadataDoesNotAffectPixels( self, ext, overrideMetadata = {}, metadat
 
 		afterMetadata = after["out"]["metadata"].getValue()
 		beforeMetadata = before["out"]["metadata"].getValue()
-		expectedMetadata = r["out"]["metadata"].getValue()
+		expectedMetadata = d["out"]["metadata"].getValue()
```

Comment:
same here, it seemed reasonable to use the metadata from the node that deletes the insignificant one